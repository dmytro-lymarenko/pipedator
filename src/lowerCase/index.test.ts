import { lowerCase } from './index';

describe('lowerCase()', () => {
	it('should validate a string in lowercase', () => {
		expect(lowerCase().validate('')).toBeNull();
		expect(lowerCase().validate('test this string')).toBeNull();
	});

	it('should fail', () => {
		expect(lowerCase().validate('test This string')).toMatchSnapshot();
	});

	it('should support custom message', () => {
		expect(lowerCase('Custom message').validate('test This string')).toMatchSnapshot();
	});
});
