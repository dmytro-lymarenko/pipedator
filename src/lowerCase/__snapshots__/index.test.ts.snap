// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lowerCase() should fail 1`] = `
Object {
  "children": null,
  "message": "Value should be in lowercase",
  "path": "",
}
`;

exports[`lowerCase() should support custom message 1`] = `
Object {
  "children": null,
  "message": "Custom message",
  "path": "",
}
`;
